test/main.sh

```diff
@@ -289,7 +289,7 @@ fi
 if [ "${1:-"all"}" = "test-shell" ]; then
   # yellow
   export PS1="\[\033[0;33mLXD-TEST\033[0m ${PS1:-\u@\h:\w\$ }\]"
-  bash --norc
+  bash --norc || true
```

Comment:
for future reference I'd encourage you to add comments to the code explaining this rather (or at least only) the commit message, as for future readers it will not be clear why this is here. We should prioritise comments in code first before (but not excluding) detailed explanations in commit messages as they may be obscured over time by future changes.